app/controllers/admin/legacy_promotional_feature_items_controller.rb

```diff
@@ -0,0 +1,92 @@
+class Admin::PromotionalFeatureItemsController < Admin::BaseController
```

Comment:
Hey, so we don't need to duplicate the controller itself. Just the tests like you've done here https://github.com/alphagov/whitehall/pull/7655/commits/b535da2272ecfbb933340f5090b869907f6936c4#diff-d54c5fbac923edc9765a4dc8052b520ed67628c4d98deb1957827f03514e9d14R1-R135

The controller itself handles all the routing of whether the user should see the [ GOV.UK Design System layout ](https://github.com/alphagov/whitehall/pull/7655/commits/b535da2272ecfbb933340f5090b869907f6936c4#diff-73f8697b8bb7cee92fcaf82cdeeae53252c0178481893a628c48e1d866ab3b6fR49-R56) and views.

Let's have a chat about the "Preview design system" permission when we're pairing later as it'll be easier to explain the context then rather than me typing out an essay. Apologies i should've gone though it earlier!

Reply:
So this controller can be completely removed. We already have this controller here https://github.com/alphagov/whitehall/pull/7655/files#diff-73f8697b8bb7cee92fcaf82cdeeae53252c0178481893a628c48e1d866ab3b6fR1-R64

We only duplicate the controller specs not the controller itself